IonDotnet/SymbolToken.cs

```diff
@@ -41,6 +46,26 @@ public SymbolToken(string text, int sid)
 
             Text = text;
             _sid = sid + 1;
+            ImportLocation = UnknownImportLocation;
+        }
+
+        /// <summary>
+        /// Create a new symbol token.
+        /// </summary>
+        /// <param name="text">Text</param>
+        /// <param name="sid">Sid</param>
+        /// <param name="importLocation">ImportLocation</param>
+        public SymbolToken(string text, int sid, ImportLocation importLocation)
```

Comment:
Is there a reason not to use the existing constructor and add `ImportLocation importLocation = default` (as is done in the IonSymbol constructor)? Then you can probably get rid of `UnknownImportLocation` since that would be denoted by `null`.

Reply:
Now using `ImportLocation importLocation = default` and got rid of `UnknownImportLocation`